### ISSUE TITLE
Clarify Body Listener Usage in Documentation

### ISSUE THREAD

**Issue by jonmchan** [FriendsOfSymfony/FOSRestBundle#854]
Can we add clarification and/or an example to the Body Listener documentation section (https://github.com/FriendsOfSymfony/FOSRestBundle/blob/master/Resources/doc/3-listener-support.md#body-listener)? I notice there is no controller example for just using the Body Listener. Is it that the Body Listener must be used with the ParamFetcher listener or the Request Body Converter Listener? If so, can this be explicitly stated in the documentation somewhere? (I apologize in advance if it is there and I just couldn't find it)

While I'm on the topic... I think the Listener Support page in general can use some cleanup. There has been many different listeners that have been added that sometimes interact with each while others can be used in alternative to each other. The other issue is that the names are all very similar - maybe we can add a TOC or definitions section that lists all the different listeners and their purpose in one sentence? Maybe we can categorize them too by what they do?

**Comment by lsmith77** [comment]
ping

**Comment by jonmchan** [comment]
Are you pinging me?

**Comment by lsmith77** [comment]
yeah .. just wondering if you are motivated to revisit the topic. that being said, the docs will soon be on symfony.com .. hoping that this will also entice more people to help on the docs.

**Comment by jonmchan** [comment]
Sorry - I no longer am working on symfony projects anymore. My work has pulled me off to RoR projects and it doesn't look like i'll be doing symfony anytime soon. So I don't really have the bandwidth to finish up the docs.
